setup.py

```diff
@@ -95,4 +95,5 @@ def get_packages(package):
     [console_scripts]
     uvicorn=uvicorn.main:main
     """,
+    project_urls={"Changelog": "https://www.uvicorn.org/release-notes/"},
```

Comment:
what about we just add the github link of the Changelog.md to this ?

Reply:
I think we can start from there. But I think we should have it on the docs as well at some point, with the reference to the PRs, it's easy to travel around.